taxi_service/urls.py

```diff
@@ -13,6 +13,7 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path("blog/", include("blog.urls"))
 """
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
The import of `debug_toolbar_urls` should be conditional based on the environment (development vs production) to avoid exposing sensitive information in production.